### ISSUE TITLE
consld8 with no switches

### ISSUE THREAD

**Issue by nomandera** [trinapicot/unraid-diskmv#3]
Based on a best guess and the output of lsof I "think" if you consld8 with no switches it will try to run /mnt/user in test mode

I have not let it complete as it would take a very very long time but regardless the raw command should probably not do anything at all other than return help.

**Comment by trinapicot** [comment]
Yep, that's a "feature" I discovered a while ago. I decided to leave it just in case someone wanted to consolidate their entire array onto one disk.

It takes a while to determine the total disk usage of '/mnt/user/' but I wouldn't say it's a very very long time. It's like clicking calculate for every share in the webgui.

I'll at least add a message to indicate what's going on: Finding disk usage of '/mnt/user/'...
I'll also reconsider disallowing this usage.

**Comment by nomandera** [comment]
I havent let it complete but I think it is particularly long for me because I run cache_dirs heavily on a number of user shares and I have a couple that are excluded from cache_dirs but contain very large file counts.

What I can say is the time is measured in many minutes. 

I would suggest an alternative default behavior would be to return the missing switches summary unless your pwd is somewhere under /mnt/user whereby you would assume running test mode from there. For me at least that seems more common than users merging at the top level.
